week-3/Homework/mandatory/1-oxygen-levels.js

```diff
@@ -8,11 +8,12 @@ To be safe, they need to land on the first unamed planet that has Oxygen levels
 
 Write a function that finds the oxygen level of the first safe planet - Oxygen between 19.5% and 23.5%
 */
-
-function safeLevels() {
-
+function safeLevels(oxygenLevels1) {
+  let safeLevel1 = oxygenLevels1.find(Element => Element > "19.5%" && Element < "23.5%");
```

Comment:
You're really close, but I don't think this is quite right.

The problem here is that you're using the greater than and less than operators with strings not numbers.

Try changing one of the values in the test to "100.5%". Does it do what you expected? How could you fix it?